app/src/main/java/com/architect/coders/mu8/repository/ComicRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.architect.coders.mu8.repository
+
+import com.architect.coders.mu8.MARVEL_PUBLIC_KEY
+import com.architect.coders.mu8.createHashCode
+import com.architect.coders.mu8.remote.RetrofitManager
+
+class ComicRepository : IComicRepository {
+    private val retrofitManager =
+        RetrofitManager()
+
+    override suspend fun getAllComics(ts: String) =
```

Comment:
Rename `ts` for `timestamp`, avoid the usage of short name variables that doesn't tell exactly what it is in an obvious way.